pom.xml

```diff
@@ -198,6 +198,7 @@
       <groupId>org.awaitility</groupId>
       <artifactId>awaitility</artifactId>
       <version>4.0.3</version>
+      <scope>test</scope>
```

Comment:
see #154

Reply:
Hmm, see also https://github.com/jenkinsci/google-compute-engine-plugin/issues/200, and https://github.com/GoogleCloudPlatform/gcp-plugin-core-java 0.3.0 still lists `awaitility` as a compile-scope dependency, which has a compile-scope dependency on `hamcrest-core` and `hamcrest-library`, so maybe this needs to be reverted? Also more generally, it looks like the `gcp-client` library being used by this plugin is abandoned, perhaps the plugin should migrate to something in https://github.com/googleapis/google-cloud-java?